pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,44 @@
+package controller
+
+import (
+	"context"
+	"github.com/openshift/cluster-policy-controller/pkg/operator"
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+const (
+	controllerName = "csr-approver-controller"
+	// Hardcoded for the Monitoring Operator:
+	// https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27
+	monitoringServiceAccountNamespace = "openshift-monitoring"
+	monitoringServiceAccountName = "cluster-monitoring-operator"
+	monitoringCertificateSubject = "/CN=monitoring"
```

Comment:
@s-urbaniak Please check those 3 items - are they correct?

Reply:
the subject seems wrong, I would expect it to match the name of the serviceaccount pulling the metrics

Reply:
Yes, the certificate requestor is cluster-monitoring-operator, the target subject is Prometheus, how about `/CN=prometheus`?

Reply:
Done